lib/components/drawer/end_drawer.dart

```diff
@@ -66,66 +69,85 @@ class LeftDrawerWidgetState extends State<LeftDrawerWidget> {
             final moderatorList = data['moderators'];
             final vipList = data['vips'];
             final viewerList = data['viewers'];
-            return CustomScrollView(
-              slivers: [
-                // const SliverSearchBarWidget(),
-                SliverList(
-                  delegate: SliverChildListDelegate(
-                    const [
-                      SizedBox(
-                        height: 64,
-                      )
-                    ],
+            final viewersListModel =
```

Comment:
Since this is the only place the viewer list is used in the app, it would be better to avoid using a model and do the filter in the widget.